migrate/20241204_vm_add_vcpus_memory.rb

```diff
@@ -2,9 +2,5 @@
 
 Sequel.migration do
   change do
-    alter_table(:vm) do
```

Comment:
Why did you change migrations

Reply:
Migrations failed on my setup because they are applied in the wrong order. I made this change to fix them in my environment, but didn't intend to commit the changes. I reverted it back now.